x-pack/packages/security-solution/features/src/cases/v1_features/kibana_sub_features.ts

```diff
@@ -55,6 +56,7 @@ export const getCasesSubFeaturesMap = ({
               delete: [APP_ID],
             },
             ui: uiCapabilities.delete,
+            replacedBy: [{ feature: CASES_FEATURE_ID_V2, privileges: ['cases_delete_v2'] }],
```

Comment:
@cnasikas any thoughts on what we want the versioning naming pattern to be? Just defaulted to `_v2` to keep it simple, but we should ideally start to set some kind of consistent standard...